usage_report/usage_report.py

```diff
@@ -211,6 +216,19 @@ def main(date, lag_days, sample, no_output, input_bucket, input_prefix, input_ve
         write_to_s3(output_bucket, s3_key_fxhealth.format(MASTER_VERSION), fxhealth_data_full)
         write_to_s3(output_bucket, s3_key_webusage.format(MASTER_VERSION), webusage_data_full)
 
+        # write annotations
+        annote_files = [
+            (f, os.path.join(ANNOTATIONS_DIR, f))
+            for f in os.listdir(ANNOTATIONS_DIR)
+            if f.endswith(ANNOTATIONS_SUFFIX)
+        ]
+
+        for filename, full_path in annote_files:
+            with open(full_path, 'r') as f:
+                data = f.read()
```

Comment:
hi @fbertsch , can you convert the annotations json into a dict before feeding it into `write_to_s3`? either here, or in the body of `write_to_s3` (check for string and if it is, convert to dict first). right now it's a string, and `write_to_s3` is expecting a dict.